### PR TITLE
Add enumerated lists to presto-admin documentation

### PR DESCRIPTION
**Pull request by cawallin**
For ease of use, add enumerated lists to the presto-admin documentation
instead of blocks of text. Fix some grammatical/style issues also.

Follow up to https://github.com/prestodb/presto-admin/pull/149

### CODE REVIEW

File: docs/installation/presto-configuration.rst [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
-based on your cluster's capacity. The default max memory for each Presto server is 16GB, but in
-an environment with less than 16GB available per node, you will have to adjust the settings
-to be lower. Similarly, if you have a lot of memory (say, 120GB/node), you may want to allocate more
+based on your cluster's capacity. The default max memory for each Presto server
+is 16 GB, but if you have a lot of memory (say, 120GB/node), you may want to allocate more
 memory to Presto for better performance.
 
-In the second case, in order to update the max memory value to 60GB per node, 
-change the line in ``/etc/opt/prestoadmin/coordinator/jvm.config`` and
+In order to update the max memory value to 60 GB per node:
```
Comment (BrendaNoonan) [inline docs/installation/presto-configuration.rst]: I like this---it was the "in the second case" that was making me look for the first case, which wasn't labeled as the first case.

File: docs/installation/presto-admin-installation.rst [hunk 2]
```diff
@@ -6,8 +6,9 @@ Presto Admin Installation
 *Prerequisites:* `Python 2.6 or Python 2.7 <https://www.python.org/downloads>`_
```
Comment (BrendaNoonan) [inline docs/installation/presto-admin-installation.rst]: Meant to mention this, but forgot! There's a mixture of form in titles in the PR (Presto Admin Installation, and later on Configuring Presto, and some others). Suggest using -ing or nouns for consistency.

**Comment by BrendaNoonan** [comment]
Just one comment about titles. Other than that, LGTM!

**Comment by cawallin** [comment]
Fixed & merged. Thanks for the review!

**Pull request closed without merging**